### DISCUSSION TITLE
How to index pubpub to dblp

### DISCUSSION THREAD

**Discussion by hk027** [knowledgefutures/pubpub#2196] [Support]
Hi, 
My name is Kalonji, 
My question is about the indexation of pubpub to dblp. 
I created some proceedings for a conference. But now , i am trying to understand how I can index pubpub to dblp. Is there any document about the indexation of pubpub to other databases  ? or is someone can have knowledge about that? 
Please assist me. 
Kalonji

**Comment by gabestein** [comment]
Answered via email, but for other folks wondering, reading the FAQ it looks to me like dblp will index HTML with good metadata on request ([instructions](https://dblp.org/faq/5210229.html)), assuming their [criteria](https://dblp.org/faq/5210119.html) is met. We're waiting to hear back on Kalonji's request.
